accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-text-rules.ts

```diff
@@ -37,9 +37,11 @@ let a11yRulesText: Rule[] = [
                 }
             }
             const ruleContext = context["dom"].node as Element;
+            //skip the rule
+            if (RPTUtil.isNodeHidden(ruleContext)) return null;
```

Comment:
This isn't a purely AT specific rule, so I don't think this should be skipped.

Reply:
removed